js/common.ts

```diff
@@ -1169,7 +1304,9 @@ async function startTransaction(
   }
 }
 
-async function startTask(db: pgPromise.IDatabase<unknown, IClient>) {
+async function startTask(
+  db: pgPromise.IDatabase<unknown, IClient> | ITask<unknown>,
+): Promise<{ t: pgPromise.IDatabase<unknown, IClient> | ITask<unknown>; endTask: () => void }> {
```

Comment:
removed Idatabase from the types and only kept ITask (on the top level types, on utility functions I often still allow both because for them it often doesn't matter)